mev_inspect/aave_liquidations.py

```diff
@@ -51,7 +51,17 @@ def get_aave_liquidations(
                 trace.transaction_hash, trace.trace_address, traces
             )
 
-            received_amount = _get_liquidator_payback(child_traces, liquidator)
+            payback_transfer = _get_liquidator_payback(child_traces, liquidator)
+
+            if payback_transfer:
+                assert isinstance(payback_transfer, ERC20Transfer)
```

Comment:
is this necessary? i'd think it'd fail down in the function

`ERC20Transfer.from_trace` inside the function should definitely return `ERC20Transfer`

Reply:
Yes, mypy